newrelic/hooks/hybridagent_opentelemetry.py

```diff
@@ -46,17 +95,17 @@ def wrap_set_tracer_provider(wrapped, instance, args, kwargs):
 
 
 def wrap_get_tracer_provider(wrapped, instance, args, kwargs):
-    settings = global_settings()
-
-    if not settings.otel_bridge.enabled:
-        return wrapped(*args, **kwargs)
-
     # This needs to act as a singleton, like the agent instance.
     # We should initialize the agent here as well, if there is
     # not an instance already.
     application = application_instance(activate=False)
```

Comment:
Change this to grab the application with activate=True and then activate if it's not active.